server/org.eclipse.emfcloud.ecore.backend-app/org.eclipse.emfcloud.ecore.backend.app/src/org/eclipse/emfcloud/ecore/backend/app/GenModelApplication.java

```diff
@@ -0,0 +1,65 @@
+package org.eclipse.emfcloud.ecore.backend.app;
+
+import java.io.File;
+import org.eclipse.emf.codegen.ecore.genmodel.GenModel;
+import org.eclipse.emf.codegen.ecore.Generator;
+import org.eclipse.equinox.app.IApplication;
+import org.eclipse.equinox.app.IApplicationContext;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.EPackage;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+
+public class GenModelApplication implements IApplication {
+
+	@Override
+	public Object start(IApplicationContext context) throws Exception {
+		Generator codegen = new Generator();
+		String[] args = getArgs(context);
```

Comment:
I would extract the args to individual variables, and stop using the string-array as early as possible. Manipulating arg[0] inside the code isn't really readable